control-plane/cmd/kafka-broker-controller/main.go

```diff
@@ -34,28 +35,33 @@ const (
 )
 
 func main() {
-	brokerEnvConfigs := broker.EnvConfigs{}
 
-	if err := envconfig.Process("", &brokerEnvConfigs); err != nil {
-		log.Fatal("cannot process environment variables", err)
+	brokerEnv, err := config.GetEnvConfig("BROKER")
```

Comment:
We should prefix special knative things with `K_`

Reply:
I am not sure it is a good idea to call envconfig.Process on each get accessor.

Reply:
What do you mean by special? 

I see those configurations just like `SYSTEM_NAMESPACE` (which is part of them) which hasn't a `K_` prefix. :confused: 

> I am not sure it is a good idea to call envconfig.Process on each get accessor.

How do you set the same set of configurations with different prefixes without duplicating the struct?  We have the same set of configurations for two reconcilers with different values, `BROKER` and `SINK` are prefixes to distinguish the two: https://github.com/knative-sandbox/eventing-kafka-broker/pull/134/files#diff-2e77fe22a128ec539d95f2e10c96eff6R60-R88,